### DISCUSSION TITLE
Consider adding a Rust vs Noir comparison section to docs

### DISCUSSION THREAD

**Discussion by critesjosh** [noir-lang/noir#3112] [Ideas]
### Problem

Noir is heavily inspired by Rust. It has many of the same language features, but it is also missing many.

The features of Noir are changing quickly and the differences are probably vast. Something like this may not be worth it, but is at least worth considering.

### Happy Case

Someone that knows Rust can quickly and easily understand how to use Noir.

### Would you like to submit a PR for this Issue?

No

**Comment by Savio-Sou** [comment]
Good call. Off the top of my mind:
- Nargo vs Cargo
- `Field`
